tools/topology/topology2/platform/intel/sdw-amp-feedback.conf

```diff
@@ -0,0 +1,64 @@
+Object.Dai {
+	ALH."515" {
+		id 		$SDW_AMP_IN_BE_ID
+		direction	"capture"
+		name		$SDW_SPK_IN_STREAM
+		default_hw_conf_id	1
+		rate			48000
+		channels		2
+
+		Object.Base.hw_config."ALH515" {
+			id	1
+		}
+	}
+}
+
+Object.Pipeline {
+	passthrough-capture.30 {
```

Comment:
can we remove references to passthrough, the feedback is likely to be split and provided to the smart-amp algorithm

Reply:
> can we remove references to passthrough, the feedback is likely to be split and provided to the smart-amp algorithm

@plbossart If there is a smart-amp widget, what we need to do is to add a route from Copier.ALH.31.1 to the smart-amp widget. Copier.ALH.31.1 itself can work as a demux